metadata-ingestion/source_docs/bigquery.md

```diff
@@ -0,0 +1,127 @@
+# BigQuery
+
+To install this plugin, run `pip install 'acryl-datahub[bigquery]'`.
+
+This plugin extracts the following:
+
+- List of databases, schema, and tables
+- Column types associated with each table
+
+```yml
+source:
```

Comment:
Ditto for this one. This is a big overwhelming with all the text mixed into yaml. How can we improve?